packages/web-components/fast-foundation/src/slider/slider.ts

```diff
@@ -216,7 +222,7 @@ export class Slider extends FormAssociated<HTMLInputElement>
      */
     public disconnectedCallback(): void {
         this.removeEventListener("keydown", this.keypressHandler);
-        this.removeEventListener("mousedown", this.clickHandler);
+        this.removeEventListener("mousedown", this.handleMouseDown);
         this.thumb.removeEventListener("mousedown", this.handleThumbMouseDown);
```

Comment:
should we also remove the touchstart event listener?

Reply:
doh! removing